pkg/didcomm/crypto/legacy/authcrypt/encrypt.go

```diff
@@ -83,6 +69,7 @@ func (c *Crypter) buildEnvelope(nonce, payload, cek []byte, protected *protected
 
 	protectedB64 := base64.URLEncoding.EncodeToString(protectedBytes)
 
+	// TODO consider: does cek need to be kept secret in wallet?
```

Comment:
Issue number?

Reply:
I'll remove TODO, we did consider, it's not necessary.